integration-tests/tests/src/utils/import-app.ts

```diff
@@ -25,7 +25,7 @@ export type ErrorResponse = { message: string; appId: never };
 export type ImportResponse = { appId: string; message: never };
 export type Response = ImportResponse | ErrorResponse;
 
-function getUrls() {
+export function getUrls() {
```

Comment:
We should probably move this into its own file.  Now that `getUrls` is an export, it really doesn't have much to do with `import-app`.  Alternatively, we could consider setting defaults on the environment and doing away with this function altogether.

@JacobOscarGunnarsson This is just a refactor consideration, you can still merge this without said change, but maybe write a todo